kratos/tests/materials_with_subproperties.json

```diff
@@ -0,0 +1,110 @@
+{
+    "properties" : [{
+        "model_part_name" : "Inlets",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "HyperElastic3DLaw",
+                "todo_name" : "GeneralParallelConstitutiveLaw",
+                "sub_properties_indexes"   : [ 11,  12,  13 ],
```

Comment:
I would avoid storing the "sub_properties_index" in the CL. We can assume that indices are 0,1,2...

this is important since this array will have to be stored for each CL

Reply:
Before changing that I prefer to ear the opinion of the rest

Reply:
In fact it is not stored in an array, I use a map, look the other PR (the rule of mixtures)